src/test/java/com/dabsquared/gitlabjenkins/publisher/TestUtility.java

```diff
@@ -130,8 +131,8 @@ static String formatNote(AbstractBuild build, String note) {
 
     static <P extends MergeRequestNotifier> P preparePublisher(P publisher, AbstractBuild build) {
         P spyPublisher = spy(publisher);
-        MergeRequest mergeRequest =
-                new MergeRequest(MERGE_REQUEST_ID, MERGE_REQUEST_IID, "", "", "", PROJECT_ID, PROJECT_ID, "", "");
+        MergeRequest mergeRequest = new MergeRequest(
+                MERGE_REQUEST_ID, MERGE_REQUEST_IID, MERGE_COMMIT_SHA, "", "", "", PROJECT_ID, PROJECT_ID, "", "");
```

Comment:
```suggestion
                new MergeRequest(MERGE_REQUEST_ID, MERGE_REQUEST_IID, MERGE_COMMIT_SHA, "", "", "", PROJECT_ID, PROJECT_ID, "", "");

```

Reply:
This update was added by some linter. The CLI told me running `mvn spotless:apply` and I obeyed.

Reply:
No worries, I am still getting used to these `spotless` changes.

Reply:
Is there anything else I should do?

Reply:
None that I know of. I will pull the code locally and inspect it on my IDE later today before I merge it. But so far the code looks okay. I will probably be able to merge this PR if everything checks out and cut a new release later tonight.